Weather/Weather/Controllers/CityViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  SelectName.swift
+//  Weather
+//
+//  Created by Sergii Lisnyi on 5/17/18.
+//  Copyright © 2018 Sergii Lisnyi. All rights reserved.
+//
+import SwiftyJSON
+import UIKit
+
+class CityViewController: UIViewController {
+ 
+    @IBOutlet weak var enterLabel: UILabel!
+    @IBOutlet weak var nameLabel: UITextField!
+    var delegate: PageViewController?
+    var modelCity = CityModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    @IBAction func selectButtonTapped(_ sender: UIButton) {
+        let nameCity = nameLabel.text!
+        isValidate(nameCity: nameCity)
```

Comment:
`validate` or `valid`

Reply:
Also, I may assume that you store cityName in `modelCity`, so it's better to take data from model - not from UI presentation